dice-where-downloader-lib/src/main/java/technology/dice/dicewhere/downloader/destination/local/LocalFileAcceptor.java

```diff
@@ -39,6 +40,10 @@ public StreamConsumer<Void> getStreamConsumer(
         LOG.debug("Destination directory already exists");
       }
       Files.copy(stream, destination, StandardCopyOption.REPLACE_EXISTING);
+      if ((!noMd5Check) && (!originalFileMd5.matches(stream.md5()))) {
+        LOG.error("MD5 mismatch. Deleting destination file");
+        Files.delete(destination);
```

Comment:
just wondering if this is going to be a problem for id, because in line 42 we do replace existing and then delete , so basically we do not have any file in the destination and in id after we download we check if the expected file exists https://github.com/DiceTechnology/dce-id/blob/qa/dce-id-service/src/main/java/com/imggaming/dce/id/authenticate/geoip/DiceWhereDownloaderScheduleJob.java#L342 🤔